ros/wavemap_ros/src/tf_transformer.cc

```diff
@@ -20,6 +20,14 @@ bool TfTransformer::waitForTransform(const std::string& to_frame_id,
                               sanitizeFrameId(from_frame_id), frame_timestamp);
 }
 
+bool TfTransformer::lookupLatestTransform(const std::string& to_frame_id,
+                                          const std::string& from_frame_id,
+                                          Transformation3D& transform) {
+  ros::Time latest{};
```

Comment:
Is this temporary needed instead of being part of the call?

Reply:
I thought that giving it a name would make it a bit clearer. But it should already be clear from context and the name of `lookupLatestTransform`, so I'll move it into the call.